client/ayon_core/hosts/maya/api/lib.py

```diff
@@ -1857,7 +1880,16 @@ def get_container_members(container):
         if ref.rsplit(":", 1)[-1].startswith("_UNKNOWN_REF_NODE_"):
             continue
 
-        reference_members = cmds.referenceQuery(ref, nodes=True, dagPath=True)
+        try:
+            reference_members = cmds.referenceQuery(ref,
+                                                    nodes=True,
+                                                    dagPath=True)
+        except RuntimeError:
+            # Ignore reference nodes that are not associated with a
+            # referenced file on which `referenceQuery` command fails
```

Comment:
is it possible to have a reference node without an associated filepath ?

Reply:
I believe it is technically possible, but this maya bug isn't exactly the same. I'll look into whether I can create a reproducable scene.

Reply:
See https://github.com/ynput/ayon-core/pull/321#discussion_r1557419625